tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Controllers/UserController.cs

```diff
@@ -127,23 +127,31 @@ IOptions<GeneralSettings> generalSettings
             Type = typeof(Users))]
         [SwaggerResponseRemoveDefaults]
         public IActionResult UserProfilePicture([FromBody]Client client)
-        {    
-            #region Error Checking                
-            GenericResponseVM genericResponse = null;
-
-            if (client == null && string.IsNullOrWhiteSpace(client.Url))
+        {
+            try
             {
-                genericResponse = new GenericResponseVM()
+                #region Error Checking                
+                GenericResponseVM genericResponse = null;
+
+                if (client == null && string.IsNullOrWhiteSpace(client.Url))
                 {
-                    Value = errorSettings.MessageNoInputs,
-                    Code = HttpStatusCode.BadRequest.ToString(),
-                    IsError = true
-                };
-                return matterCenterServiceFunctions.ServiceResponse(genericResponse, (int)HttpStatusCode.OK);
+                    genericResponse = new GenericResponseVM()
+                    {
+                        Value = errorSettings.MessageNoInputs,
+                        Code = HttpStatusCode.BadRequest.ToString(),
+                        IsError = true
+                    };
+                    return matterCenterServiceFunctions.ServiceResponse(genericResponse, (int)HttpStatusCode.OK);
+                }
+                #endregion
+                var userInfo = userRepositoy.GetUserProfilePicture(client);
+                return matterCenterServiceFunctions.ServiceResponse(userInfo, (int)HttpStatusCode.OK);
+            }
+            catch(Exception ex)
+            {
+                return matterCenterServiceFunctions.ServiceResponse("Error in getting user profile picture", (int)HttpStatusCode.OK);
```

Comment:
When there is multi factor authentication issue, we are returning OK. Once the solution is found, we will return the error response to the user